satellite/admin-back-office-v1.md

```diff
@@ -0,0 +1,554 @@
+  [Oauth2 proxy](https://oauth2-proxy.github.io/)):
+  - AP1: https://ap1.admin.storj.tools
+  - EU1: https://eu1.admin.storj.tools
+  - US1: https://us1.admin.storj.tools
+
+Token secret authentication allows to access al the API endpoints. Allowed groups authentication
+only to:
+- Read user's information.
+- Read/Update user's limits.
+- Freeze/Unfreeze user's account.
+- Unwarn user.
+- Read/Update project's limits.
+
+This current web interface is a visualization of the API. An API isn't an interface for
+non-technical people, as a result, only production owners use the interface or directly the API.
+
+### Goals
+
+We want to have an intuitive Web interface that can be used by people with different backgrounds
+(e.g. Customer support, finance, sales) and becomes it the administration back office of the
+satellite for managing users' accounts.
+
+Back office has a fixed set of access levels. Every access level is associated with a list of
+groups. Users are authorized to specific access level depending to the groups they belong.
+
+Back office keeps an audit trail of all performed operations.
+
+Back office can operates without relying in any resource managed by Storj. Any satellite operator
+with or without any relation with Storj will be able to use it.
+
+### Approach / Design
+
+#### Requirements
+
+##### Operations
+
+1. Account:
+  1. View information: ID, email, full name, paid/free tier, suspended?, user agents, Multi-factor
+     authentication enabled/disabled, data placement restriction (geo fence), list of projects,
+     limits (download, storage, number of segments, number of projects allowed).
+  2. Change email address.
+  3. Disable Multi-factor authentication.
+  5. Set limits (download, storage, number of segments, number of projects allowed).
+  6. Suspend "disable upload/download".
+  7. Re-activate  "re-enable upload/download".
+  8. Set user agent.
+  9. Set data placement restriction (geo fence).
+  10. Remove data placement restriction (geo fence).
+  11. Delete account:
+    1. When the account is clean: no API keys, no data in any of the projects, no unpaid invoices).
+    2. When the account isn't clean (delinquent accounts). It include to deletes all the data
+       associated to the account.
+2. Project
+  1. View information: ID, name, creation time, limits (download, storage), user agent, data
+     placement restriction (geo fence),  stats (total used storage, total used download bandwidth,
+     total segments).
+  2. Set user agent.
+  3. Send invitation to grant access to another account.
+  4. Set data placement restriction (geo fence).
+  5. Remove data placement restriction (geo fence).
+3. Bucket
+  1. View information: ID, name, user agent, data placement restriction (geo fence).
+  2. Set user agent.
+  3. Set data placement restriction (geo fence).
+  4. Remove data placement restriction (geo fence).
+
+##### Job stories
+
+__When a user wants to access__
```

Comment:
Lets use `######` here. If there's some github bug that doesn't allow h6, then we can remove one layer by either naming these as `Story: XYZ` or making the `Operations` and such a level higher.

Reply:
I didn't use 6 because of Github incompatibilities, I wanted to have that at the same level as _Operations_.